hs_core/models.py

```diff
@@ -2206,8 +2231,16 @@ def parse_citation_name(self, name, first_author=False):
 
         return author_name + ", "
 
+    def get_custom_citation(self):
+        """Get custom citation"""
+        try:
+            return str(self.metadata.citation.all()[0]).strip()
```

Comment:
Since this is a new metadata element, we have the opportunity to easily clean the data before saving it.  I'd prefer the strip to occur before saving, not on the read.

Reply:
I think you can use ```self.metadata.citation.all().first()``` rather than ```self.metadata.citation.all()[0]```

Reply:
first() is being used